test/network_tests/test_basics.py

```diff
@@ -1,11 +1,12 @@
 #!/usr/bin/env python
 
+from __future__ import absolute_import
```

Comment:
Why is this needed here?

Same question below.

Reply:
https://www.python.org/dev/peps/pep-0328/#rationale-for-absolute-imports